domain-auth-methods.md

```diff
@@ -0,0 +1,118 @@
+	-	Domain Authorization Document must substantiate that the communication came from the Domain Contact
+	-	CA must verify that the Domain Authorization Document was either
+		-	Dated on or after the date of the domain validation request or
+		-	The WHOIS data has not materially changed since any previously provided Domain Authorization Document for the Domain Name Space
+
+6. **Agreed‐Upon Change to Website** - By confirming one of the following under “/.well-known/pki-validation” directory or other path registered with IANA for the purpose of Domain Validation, on the Authorization Domain Name that is accessible by the CA via HTTP/HTTPS over an Authorized Port
+	-	Presence of Required Website Content contained in the content of a file or on a web page in the form of a meta tag.
+	-	Presence of the Request Token or Request Value contained in the content of a file on a webpage in the form of a meta tag.
+	-	When a Random Value is used:
+		-	Random Value must be unique to the certificate request.
+		-	Random Value SHALL remain valid for no more than 30 days from its creation
+	-	Examples of Request tokens include but are not limited to:
+		-	Hash of the public key
+		-	Hash of the Subject Public Key Info
+		-	Hash of a PKCS#10 CSR
+		-	May be concatenated with a timestamp or other data
+		-	Challenge password in the creation of a CSR with OpenSSL to ensure uniqueness
+7. [ Deleted ]
+8. [ Deleted ]
+9. [ Deleted ]
+
+10. **TLS Using a Random Number (embedded in a Certificate)** - By confirming the presence of a Random Value within a Certificate on the Authorization Domain name which is accessible by the CA via TLS over an Authorized Port
+
+11. **Other methods** - 
+
+> "The CA SHALL confirm that, as of the date the Certificate issues,
+> either the CA or a Delegated Third Party has validated each
+> Fully‐Qualified Domain Name (FQDN) listed in the Certificate by using
+> any method of confirmation, provided that the CA maintains documented
+> evidence that the method of confirmation establishes that the
+> Applicant is the Domain Name Registrant or has control over the Fully
+> Qualified Domain Name (FQDN)."
+
+## Federal Government Method Selection
+
+The newly added "Other" option is now on the table, but it's unclear exactly what we would define and if it would continue to be acceptable over time. Therefore, the clear choice is #6 **Agreed‐Upon Change to Website**. Option #5 does not work well in the Government environment because it concentrates the effort on individuals rather than distributing the it to system administrators. Option #10  is not viable as that creates additional effort and more room for error than #5.
+
+The only decision point left is to determine exactly how to implement option #6, i.e. which request token to use. However, that decision is ultimately an implementation detail that has no effect on the 
```

Comment:
the implementation details are important - to determine what the issuing CA requirements are, what the govt wide policy impacts are (not **certificate** policy), and the impacts on automation